tox.ini

```diff
@@ -140,6 +140,14 @@ deps =
 commands =
     bash -c "yamllint eve/main.yml salt/metalk8s/defaults.yaml"
 
+[testenv:lint-sls]
+description =
+  Lint Salt SLS using salt-lint.
```

Comment:
We need a linter for `tox.ini` also, there is an indentation problem here. :)

Reply:
:smile: oh !